### PR TITLE
Rg diettracker2

### PR DESCRIPTION
**Pull request by rgoolishian**
Created DatabaseConnector and DbBuilder classes and test classes

### CODE REVIEW

**Comment by sparky005** [comment]
This looks good to me to, but you're right, I think the pom.xml changes are causing a lot of conflicts right now. This commit and my earlier commit are both showing conflicts, I have a feeling it'll be the pom file again.

File: src/main/java/edu/uml/diet/DatabaseConnector.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+    public String getJDBC_DRIVER() {
+        return JDBC_DRIVER;
+    }
+
+    public void setJDBC_DRIVER(String JDBC_DRIVER) {
+        this.JDBC_DRIVER = JDBC_DRIVER;
+    }
+
+    public String getHOST_URL() {
```
Comment (smarks) [inline src/main/java/edu/uml/diet/DatabaseConnector.java]: camel methods names getHostUrl() 

http://www.oracle.com/technetwork/java/codeconventions-150003.pdf

File: src/main/java/edu/uml/diet/DbBuilder.java [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+            while(resultSet.next()){
+                String databaseName = resultSet.getString(1);
+                if (databaseName.equals(this.dbName)){
+                    return true;
+                }
+            }
+            resultSet.close();
+        }
+        finally{
+            if(stmt!=null)
```
Comment (smarks) [inline src/main/java/edu/uml/diet/DbBuilder.java]: whole word variable names

File: src/main/java/edu/uml/diet/DbUserServices.java [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+public class DbUserServices implements PersistanceUserServices {
+
+    private String DbName = "diettracker";
+    private String JDBC_DRIVER = "com.mysql.jdbc.Driver";
+    private String HOST_URL = "jdbc:mysql://localhost/";
+    private String DB_URL = "jdbc:mysql://localhost/" + DbName;
+    private String USER = "root";
+    private String PASS = "PASSWORD";
+    public DatabaseConnector databaseConnector = new DatabaseConnector(JDBC_DRIVER,HOST_URL,DB_URL,USER,PASS);
+    public DbBuilder dbBuilder = new DbBuilder(databaseConnector,"diettracker");
```
Comment (smarks) [inline src/main/java/edu/uml/diet/DbUserServices.java]: properties

File: src/main/java/edu/uml/diet/DbUserServices.java [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+    private String TableName = "users";
+
+
+    /**
+     * Method to add new user to database
+     *
+     * @param username
+     * @param password
```
Comment (smarks) [inline src/main/java/edu/uml/diet/DbUserServices.java]: javadoc return

**Comment by rgoolishian** [comment]
Completed target changes for this weekend.

**Comment by sparky005** [comment]
Can we merge this? I'm trying to get the login stuff working but the database stuff in master is old

**Pull request merged**